### PR TITLE
Bugfix: Skip tests for tools not being built

### PR DESCRIPTION
**Pull request by luke-jr**

### CODE REVIEW

**Comment by fanquake** [comment]
Not sure what you expect to be done here. Even if Github, Appveyor and DrahtBot are all wrong about there being a conflict, if this PR can't be merged using the [merge-script](https://github.com/bitcoin-core/bitcoin-maintainer-tools/blob/master/github-merge.py) (it currently can't), it's not going to be merged. The fact that no CI is running is also a blocker. I'd suggest rebasing on a master more recent than April 2020.

**Comment by MarcoFalke** [comment]
DrahtBot and appveyor use the GitHub API, so this is mostly a GitHub bug.

**Comment by MarcoFalke** [comment]
My suggestion would be to close this pull request, since it is not possible to merge with the merge script without modifying it.

**Comment by luke-jr** [comment]
Please open a new, separate issue for bugs in your merge script/bots. It's off-topic here.

This PR still merges cleanly to master. 0.20, and 0.21.

**Comment by kristapsk** [comment]
Btw, I'm using the same recent `github-merge.py` script for JoinMarket, it also fails with empty pull request description.

**Comment by achow101** [comment]
Are you still working on this?

File: test/config.ini.in [hunk 1]
```diff
@@ -20,6 +20,8 @@ RPCAUTH=@abs_top_srcdir@/share/rpcauth/rpcauth.py
 @USE_BDB_TRUE@USE_BDB=true
 @BUILD_BITCOIN_CLI_TRUE@ENABLE_CLI=true
 @BUILD_BITCOIN_WALLET_TRUE@ENABLE_WALLET_TOOL=true
+@BUILD_BITCOIN_TX_TRUE@ENABLE_UTIL_TX=true
+@BUILD_BITCOIN_UTIL_TRUE@ENABLE_UTIL_UTIL=true
```
Comment (fanquake) [inline test/config.ini.in:24]: We already have `ENABLE_BITCOIN_UTIL`, so we should use that.

**Review by theuni — CHANGES_REQUESTED** [review]
@luke-jr If you're still interested in this, I agree with the previous reviewers:
- PR needs a description, if only for posterity
- As @fanquake mentioned, `ENABLE_BITCOIN_UTIL` could be used here.

I'm not at all familiar with the test framework, but this looks a lot like a re-implementation of something that's already cleanly abstracted. What exactly isn't getting correctly skipped?

**Comment by fanquake** [comment]
> May want to close this, given the ongoing cmake transition obsoletes it either way?

Think I agree. If the (asked-for) review/questions are addressed, can still be re-opened.

**Pull request closed without merging**